control/config.txt

```diff
@@ -839,4 +839,4 @@ logToFile_Errors
 logToFile_Messages
 logToFile_Warnings
 history_max 50
-gameGuard 0
+gameGuard 1
```

Comment:
this should not be here

Reply:
We will remove this when we merge with the master. I am planning to remove this setting from config.txt